lib/utilities/collectResponseHeaders.js

```diff
@@ -45,6 +45,9 @@ function collectResponseHeaders(objectMD, corsHeaders, versioningCfg,
         responseMetaHeaders['x-amz-server-side-encryption-aws-kms-key-id']
             = objectMD['x-amz-server-side-encryption-aws-kms-key-id'];
     }
+    if (objectMD['accept-ranges']) {
```

Comment:
Why is this part of `objectMD`? For me it's a server-side capability that is not related to object internal metadata.